.gitignore

```diff
@@ -26,6 +26,9 @@ build/Release
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
 
+#lock file
+package-lock.json
```

Comment:
the lock file is not gitignored with good reason. it provides a way to make reliable builds. you dont want some upstream deps to get upgraded and your build doesnt work anymore

Reply:
woops will fix that